NitroxPatcher/Patches/Dynamic/StasisSphere_Shoot_Patch.cs

```diff
@@ -0,0 +1,26 @@
+using System.Reflection;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
+using NitroxModel_Subnautica.DataStructures;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public sealed partial class StasisSphere_Shoot_Patch : NitroxPatch, IDynamicPatch
+{
+    private static readonly MethodInfo TARGET_METHOD = Reflect.Method((StasisSphere t) => t.Shoot(default, default, default, default, default));
+
+    private static ushort LocalPlayerId => Resolve<LocalPlayer>().PlayerId;
```

Comment:
Don't know if this is needed. As it's technically a method nothing is cached

Reply:
It was just there for reading purposes